JavaScript/Microsoft.ApplicationInsights.JavaScript.sln

```diff
@@ -1,13 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Never mind, I saw the comment that you put in saying this will be removed as part of subsequent changes.